### ISSUE TITLE
Question on Improving Reconstruction Quality with Custom Dataset

### ISSUE THREAD

**Issue by yqytomorrow** [DekuLiuTesla/CityGaussian#196]
Thank you very much for your excellent and impressive work, as well as for making the code publicly available to the community.

I am currently using your method to train on my own custom dataset and would like to kindly ask for your advice on how to further improve the reconstruction quality.

**My current experimental setup is as follows:
Image resolution: approximately 4K
Number of images: around 800
Down-sampling factor: 1.0 (i.e., no down-sampling)
Spherical harmonics degree: sh_degree = 3**

The images are collected from a real-world outdoor scene with relatively sufficient viewpoint coverage. However, after training and rendering, I observe that the reconstructed Gaussian results exhibit a certain degree of geometric unevenness and noise, as shown in the figure.
<img width="2061" height="1128" alt="Image" src="https://github.com/user-attachments/assets/6727c46d-628c-41dd-993b-511d48cdf06c" />
<img width="2058" height="1095" alt="Image" src="https://github.com/user-attachments/assets/e4b70a23-414f-4a75-be50-ef2afe1850f4" />

Therefore, I would like to ask whether you might have any recommendations or experience to share regarding the following aspects:

**Data acquisition strategies** (e.g., camera motion patterns, viewpoint density and overlap, capture trajectories, lighting conditions);

**Data pre-processing steps** (e.g., image pre-processing, depth initialization strategies, whether masking for sky regions or dynamic objects is necessary);

**Parameter settings** (e.g., the choice of sh_degree, learning rates, or other sensitive hyperparameters);

Common but easily overlooked issues when training on high-resolution custom datasets.

Any insights or practical suggestions based on your experience would be greatly appreciated and would be very helpful for further improving my results.

Thank you again for your excellent work and for your valuable time. I look forward to your reply.

**Comment by yqytomorrow** [comment]
The images from my real-world captured dataset are shown in the figure. <img width="1278" height="963" alt="Image" src="https://github.com/user-attachments/assets/6bae60dd-df2a-45b2-85a9-85b2d7a02d2b" />

<img width="1278" height="945" alt="Image" src="https://github.com/user-attachments/assets/9f86d621-62fa-45f1-9d8d-fdc5ee5f6274" />

**Comment by DekuLiuTesla** [comment]
@yqytomorrow You can try the practice in CityGS-v2 with Depth-Anything-V2 for depth initialization, and in our experiments, the hyperparameters used in datasets like "Russian" generalize well to other scenes. For the aerial view, we do not mask the sky, and we found that the effect of dynamic objects will be averaged and can be ignored. For capturing, we recommend at least 1/3 overlap for the nearby views, and constant light conditioning is an ideal choice.
